### ISSUE TITLE
Congenital CMV infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6159]
**Preferred term label:**
Congenital CMV infection 

**Synonyms**
Congenital cytomegalovirus infection 

**Definition (free text, please give PubMed ID)**
A congenital CMV infection of the newborn can follow either a primary or recurrent maternal infection. Jaundice, petechiae, and hepatosplenomegaly are the most frequently noted clinical triad in symptomatic infants. PMID: 29503048 

**Parent term (use hpo.jax.org/app)**
Severe infection and Persistent CMV viremia

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:294

**Comment by pnrobinson** [comment]
I think this is best characterized as a disease.  however, the history of congenital CMV is important.
HPO has a relatively new section for Past Medical history. 
https://hpo.jax.org/app/browse/term/HP:0032443
I added this term there.


History of congenital CMV infection
A congenital cytomegalovirus (CMV) infection of the newborn can follow either a primary or recurrent maternal infection. Jaundice, petechiae, and hepatosplenomegaly are the most frequently noted clinical triad in symptomatic infants. Affected infants may develop permanent disabilities such as hearing loss, vision loss, motor and cognitive deficits.
